.travis.yml

```diff
@@ -76,4 +76,21 @@ jobs:
         - coverage combine
         - coverage report
         - coverage xml -i -o coverage_xml.xml
-        - codecov -X gcov --file coverage_xml.xml
\ No newline at end of file
+        - codecov -X gcov --file coverage_xml.xml
+
+    - &scons_build_job
+        name: SCons Build
```

Comment:
this jobs properties are all indented 2 more spaces, could we get them all aligned?

Reply:
done